spk/comskip/Makefile

```diff
@@ -0,0 +1,40 @@
+SPK_NAME = comskip
+SPK_VERS = 0.81.093
+SPK_REV = 1
+SPK_ICON = src/comskip.png
+BETA = 1
+
+REQUIRED_DSM = 5.0
+
+DEPENDS = cross/argtable cross/ffmpeg cross/$(SPK_NAME)
```

Comment:
`cross/$(SPK_NAME)` should be enough here since `cross/comskip` alreday depends on argtable and ffmpeg